### PR TITLE
support hmdb51

### PR DESCRIPTION
**Pull request by loveunk**

### CODE REVIEW

**Comment by yjxiong** [comment]
Looks good to me. @zhaoyue-zephyrus please review and merge.

File: DATASET.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The supported datasets are listed below.
 We provide shell scripts for data preparation under the path `$MMACTION/data_tools/`.
 To ease usage, we provide tutorials of data deployment for each dataset.
 
+- [HMDB51](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/): See [PREPARING_HMDB101.md](https://github.com/open-mmlab/mmaction/tree/master/data_tools/hmdb51/PREPARING_HMDB101.md)
```
Comment (zhaoyue-zephyrus) [inline DATASET.md]: should be "HMDB51"

File: data_tools/hmdb51/extract_rgb_frames.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/bash env
+
+num_gpu=($(nvidia-smi -L | wc -l))
+num_worker=${num_gpu}
+
+cd ../
+python build_rawframes.py ../data/hmdb51/videos/ ../data/hmdb51/rawframes/ --level 2  --ext avi --num_gpu ${num_gpu} --num_worker ${num_worker}
+echo "Raw frames (RGB only) generated for train and val set"
+
+cd ava/
```
Comment (zhaoyue-zephyrus) [inline data_tools/hmdb51/extract_rgb_frames.sh]: should be "cd hmdb51/"

**Comment by zhaoyue-zephyrus** [comment]
Hi @loveunk , Thanks for your contribution! Could you kindly see my comments and revise accordingly?

**Comment by loveunk** [comment]
@zhaoyue-zephyrus Thanks for your review and comments. I have fixed the issues and rebased the commits.

**Comment by yjxiong** [comment]
@zhaoyue-zephyrus 
Please review the PR after the ECCV churn.

**Pull request merged**